packages/realm/RealmJS.podspec

```diff
@@ -5,6 +5,10 @@ package = JSON.parse(File.read(File.expand_path('package.json', __dir__)))
 
 app_path = File.expand_path('../..', __dir__)
 
+build_realm_core = ENV["REALM_BUILD_CORE"] == "1"
```

Comment:
Since this is only being used in a single place, would it make sense to inline it?